pages/blog/_posts/2020-01-28-Thanos-infinite-metrics.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blogpost
+permalink: /blog/metrics-unlimited-thanos/
+title: "Towards Observability nirvana: Infinite metric retention with Thanos"
+date: 2020-01-28
+tags: infrastructure observability prometheus thanos kubernetes
+author: <a href="https://www.linkedin.com/in/hihrig/">Holger Ihrig</a>
+---
+In nowadays DevOps world our industry is relying on the ability to observe and monitorize our infrastructure and 
+services quite a bit. Working Group Two is no exception at this and as we are operating in the TelCo space
```

Comment:
~quite a bit~

Reply:
Working Group Two is no exception here, and

Reply:
Done